exonum-java-binding-common/src/test/java/com/exonum/binding/common/proofs/map/UncheckedFlatMapProofTest.java

```diff
@@ -94,10 +94,10 @@ void mapProofWithSeveralLeafsShouldBeCorrect() {
         new MapEntry(fourthKey, THIRD_VALUE)
     );
 
-    Set<MapEntry> actualCheckedEntriesList = checkedMapProof.getEntries();
+    Set<MapEntry> actualCheckedEntriesSet = checkedMapProof.getEntries();
 
     assertThat(
-        actualCheckedEntriesList,
+        actualCheckedEntriesSet,
```

Comment:
Why are `expectedEntriesList` the same as leaves? Is it intended?

(I'd also drop the suffix specifying the type)